### PR TITLE
Fix roll call handling

### PR DESCRIPTION
**Pull request by tuomas-p**

### CODE REVIEW

File: be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandler.scala [hunk 1]
```diff
@@ -66,7 +66,7 @@ case object RollCallHandler extends MessageHandler {
               //a closeRollCall is always sent in the lao's main channel so we are allowed to do this
               val socialChannel: String = generateSocialChannel(rpcMessage.getParamsChannel, head)
               val ask: Future[GraphMessage] = (dbActor ? DbActor.CreateChannel(Channel(socialChannel), ObjectType.CHIRP)).map {
-                case DbActorWriteAck() => createAttendeeChannels(tail, rpcMessage)
+                case DbActorAck() => createAttendeeChannels(tail, rpcMessage)
```
Comment (BadrTad) [inline be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandler.scala:69]: I suppose that this case was not handled in the test suite, Would you please cover it ?

**Review by BadrTad — COMMENTED** [review]
Would be great to cover this in the tests

File: be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandler.scala (hunk 1 above)
Comment (tuomas-p) [reply be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/RollCallHandler.scala:69]: (starting to be done in PR#701)

**Review by pierluca — APPROVED** [review]
Approving blindly.

**Pull request merged**